pallets/common/src/traits/multisig.rs

```diff
@@ -42,4 +43,9 @@ pub trait MultiSigSubTrait<AccountId> {
     /// # Arguments
     /// * `account` - AccountId to check
     fn is_multisig(account: &AccountId) -> bool;
+    /// Helper function that returns if a key is a multisig signer
+    fn is_signer(key: &AccountId) -> bool;
+
+    /// Helper function that returns if a key is a multisig signer
+    fn did_of_signer(key: &AccountId) -> IdentityId;
```

Comment:
The comment needs updating.

Reply:
This function was removed as it is no longer needed